### ISSUE TITLE
Support Maven ciManagement to identify the target build job

### ISSUE THREAD

**Issue by anchialas** [loewenfels/dep-graph-releaser#56]
Support the optional `<ciManagement>` section in project's pom.xml to identify the target build job on the CI Server, e.g.
```
    <ciManagement>
        <system>jenkins</system>
        <url>https://jenkins.foo.bar/job/my-project/</url>
    </ciManagement
```

**Comment by robstoll** [comment]
I suggest that we add such projects automatically to remoteRegex. This way one can still modify it (modify remoteRegex under Settings) without the need of making a commit. 

Since a first-win strategy applies when it comes to remoteRegex we would need to append them to the remoteRegex list. For instance: one has defined the following remoteRegex `com.example:.*#https://remote/jenkins` which means all projects with groupId `com.example` are build on `https://remote/jenkins`. Let's assume project `com.example:test` has defined `<ciManagement>` as shown in the feature request description then the resulting remoteRegex will look as follows:
```
com.example:test#https://jenkins.foo.bar/
com.example:.*#https://remote/jenkins
```

Moreover, I suggest we check what one has defined under `<system>` if does not correspond to `jenkins` then we should populate a warning for the user and ignore the `<url>`.

**Comment by anchialas** [comment]
Good suggestion. This setting should not only be applied to the `remoteRegex`, but also to `jobMapping`.

**Comment by robstoll** [comment]
resolved with bcb4873
